turbinia/jobs/bulk_extractor.py

```diff
@@ -35,7 +37,8 @@ class BulkExtractorJob(interface.TurbiniaJob):
 
   # The types of evidence that this Job will process
   evidence_input = [
-      RawDisk, GoogleCloudDisk, GoogleCloudDiskRawEmbedded, EwfDisk
+      RawDisk, GoogleCloudDisk, GoogleCloudDiskRawEmbedded, EwfDisk, Directory,
+      CompressedDirectory
```

Comment:
Since we will now have the `CompressedDirectory` evidence type here, can you also add `state.DECOMPRESSED` to the `REQUIRED_STATES` attribute of `class BulkExtractorTask` so that it will be decompressed when that evidence type is used?

Reply:
Here I'm actually quite unsure what to do as bulk-extractor is able to process CompressedDirectory on its own. We do not need to uncompress. It therefore seems like a design question: Do you want to process as much as possible using turbinia and leave everything to the third-party-tools or pass the evidence as fast and unprocessed to the tools and let them do the job?
Just let me know how you want it.

Reply:
@aarontp what do you suggest?